schema/src/lib.rs

```diff
@@ -211,9 +211,8 @@ impl<I: IntoSchema, P: DecimalPlacesAware> IntoSchema for fixnum::FixedPoint<I,
                 decimal_places: P::decimal_places(),
             })
         });
-        if !metamap.contains_key(&I::type_name()) {
-            I::schema(metamap);
-        }
+
+        I::schema(metamap);
```

Comment:
I think we can introduce infinite recursion this way.

Reply:
I think you are correct. I should revert these